src/dscontrib/sustainability/utils.py

```diff
@@ -0,0 +1,58 @@
+import pandas as pd
+import pyspark.sql.functions as F
+
+
+def bootstrap(df, iterations, reporting_freq=10, strata=[],
+              treatment='variant', control='control'):
+    def munge(s):
+        agg = s.groupby(["experiment_branch"] + strata).mean()
+        agg = (
+          pd.melt(agg.reset_index(), id_vars=["experiment_branch"] + strata)
+          .pivot_table(
+                    index=strata + ['variable'],
+                    columns='experiment_branch',
+                    values='value').reset_index()
+          )
+        agg['change'] = (agg[treatment] - agg[control]) / agg[control]
+        return agg
+
+    n = len(df)
+    results = munge(df)
+    results['iteration'] = -1  # true sample stats lie at -1
```

Comment:
I'd strongly prefer returning a tuple instead of prepending the point estimates.